### ISSUE TITLE
Adding Analytics wrecks Fonts

### ISSUE THREAD

**Issue by Keltin42** [vercel/analytics#78]
I have a simple next.js /app setup.  I am using google fonts following the guidance in the docs.

```javascript
import { Odibee_Sans, Ubuntu } from '@next/font/google';
import GridHeader from './GridHeader';
import styles from './globals.css';
import GridRightHeader from './GridRightHeader';
import Footer from './footer';
import ScrollToTop from '@/components/ScrollToTop';

const obidee_sans = Odibee_Sans({
    subsets: ['latin'],
    weight: ['400'],
    variable: '--font-obidee-sans',
    display: 'optional',
});

const ubuntu = Ubuntu({
    subsets: ['latin'],
    weight: ['400'],
    variable: '--font-ubuntu',
    display: 'optional',
});

export default function RootLayout({ children }) {
    return (
        <html lang="en" className={`${obidee_sans.variable} ${ubuntu.variable}`}>
            {/*
        <head /> will contain the components returned by the nearest parent
        head.jsx. Find out more at https://beta.nextjs.org/docs/api-reference/file-conventions/head
      */}
            <head />
            <body>
                <div className="app_grid" id="root">
                    <ScrollToTop />
                    <GridHeader />
                    <GridRightHeader />
                    <div id="app_grid_content" className="app_grid_content">
                        {children}
                        <Footer />
                    </div>
                </div>
            </body>
        </html>
    );
}
```

I added Analytics to my project, but the moment I add it to my layout, the fonts stop loading.  

```javascript
import { Analytics } from '@vercel/analytics/react';
import { Odibee_Sans, Ubuntu } from '@next/font/google';
import GridHeader from './GridHeader';
import styles from './globals.css';
import GridRightHeader from './GridRightHeader';
import Footer from './footer';
import ScrollToTop from '@/components/ScrollToTop';

const obidee_sans = Odibee_Sans({
    subsets: ['latin'],
    weight: ['400'],
    variable: '--font-obidee-sans',
    display: 'optional',
});

const ubuntu = Ubuntu({
    subsets: ['latin'],
    weight: ['400'],
    variable: '--font-ubuntu',
    display: 'optional',
});

export default function RootLayout({ children }) {
    return (
        <html lang="en" className={`${obidee_sans.variable} ${ubuntu.variable}`}>
            {/*
        <head /> will contain the components returned by the nearest parent
        head.jsx. Find out more at https://beta.nextjs.org/docs/api-reference/file-conventions/head
      */}
            <head />
            <body>
                <div className="app_grid" id="root">
                    <ScrollToTop />
                    <GridHeader />
                    <GridRightHeader />
                    <div id="app_grid_content" className="app_grid_content">
                        {children}
                        <Footer />
                        <Analytics />
                    </div>
                </div>
            </body>
        </html>
    );
}
`

That's literally the only change in my project.  You can just add/remove the Analytics call and the fonts flip back and forth.  Any idea what is going on?  I'm literally just following the docs for both fonts and analytics, but you put them together and it ain't no worky.

**Comment by tobiaslins** [comment]
Hey!
Have you tried it adding it before the closing `</body>`? 
```
export default function RootLayout({ children }) {
    return (
        <html lang="en" className={`${obidee_sans.variable} ${ubuntu.variable}`}>
            {/*
        <head /> will contain the components returned by the nearest parent
        head.jsx. Find out more at https://beta.nextjs.org/docs/api-reference/file-conventions/head
      */}
            <head />
            <body>
                <div className="app_grid" id="root">
                    <ScrollToTop />
                    <GridHeader />
                    <GridRightHeader />
                    <div id="app_grid_content" className="app_grid_content">
                        {children}
                        <Footer />

                    </div>
                </div>
                <Analytics />
            </body>
        </html>
    );
}
```

**Comment by Keltin42** [comment]
doh! that totally fixed it.  thank you!  :)

I don't know why I fixated on the "add it after {children}" instead of "add it right before </body>".  Not enough caffeine methinks.  

Thank you very much, sorry to bother you with something so simple!  :)

**Comment by tobiaslins** [comment]
It shouldn't break because of that tho, we'll investigate. Thanks for raising that and happy that it works now :)

**Comment by Keltin42** [comment]
Ok, that isn't a

**Comment by tobiaslins** [comment]
Hey @Keltin42 
I'm not sure if the `<Analytics/>` component component can cause that behaviour. Do you have a deployment URL where we can reproduce this? 
What next.js version are you using?

**Comment by Keltin42** [comment]
Upgraded to 13.4.1 this morning and it seems to be resolved.  I literally haven't changed anything else since, so I'm still not sure what was conflicting, but it doesn't seem to be an issue now.

**Comment by s-rd** [comment]
I'll try not to reopen this issue already, but this is happening in my Next.js site too. Adding `@vercel/analytics/next` and using the component in the layout, like in the docs. Some styles, and specifically fonts, suddenly break. Removing the one line fixes the issue.

Thought I'd add it here in case others face the same/similar issues.

**Comment by tobiaslins** [comment]
@s-rd do you see it only locally or also after deploying? A repro would help us!

**Comment by s-rd** [comment]
@tobiaslins Thanks for the super quick response! Would love to make a dummy repro but a bit short on time on this one. 

That said I think I figured it out, but it's a strange one – whenever I included the `<Analytics />` component, it seemed to result in my `reset.css` being injected _after_ my other styles. My `reset.css` was being loaded from my `/public` folder in my Next.js project, so I resolved it by moving it to `/app/styles` instead.

Perhaps a poor setup on my end, but it took a few hours to figure out the connection. Sounds like it's possibly more of an issue on Next's end rather than `@vercel/analytics`.
